tests/h/services/search_index/_queue_test.py

```diff
@@ -18,127 +20,132 @@
 MINUS_5_MIN_IN_SECS = int(MINUS_5_MIN.total_seconds())
 
 
-class TestAddSyncAnnotationJob:
-    def test_it(self, db_session, factories, queue, now):
-        annotation = factories.Annotation.build()
+class TestAddMethods:
```

Comment:
In a move designed to ensure nobody is pleased, I'm going to rename the add methods tests to `TestQueue` and leave the sync ones separate as `TestSync`. This displays two models which are common in the code (class specific and method specific). The only novelty here is to mix the two.

Why? Because it respects a seam in the code, and doesn't mix together the two sets of tests unecessarily. This will pay off if we want to refactor this into two classes one day, as the tests will not have mixed as much.